aries-site/src/examples/components/layer/LayerSideDrawerExample.js

```diff
@@ -97,11 +97,10 @@ const LayerForm = ({ setOpen }) => {
         <Box
           direction="row"
           gap="large"
-          justify="end"
           margin={{ top: 'medium', bottom: 'small' }}
         >
+          <Button label="Submit Form" primary type="submit" />
```

Comment:
The gap here is enormous. Is that what we want? Should be bumped down.

This is another instance where the Figma file doesn't seem to be using AutoLayout and proper spacing. The implementation in the file has a non-standard spacing of 41px.

Reply:
For comparison /templates/forms#filter is using medium and examples on https://design-system.hpe.design/templates/filtering?q=filt use small.